pkgs/applications/science/molecular-dynamics/raspa/data.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenvNoCC
+, gzip
+, raspa
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "raspa-data";
+  inherit (raspa) version src;
+
+  # TODO: Remove "out" from the outputs and
+  # remove the line `mkdir "$out"` from installPhase
+  # when #16182 get solved.
+  outputs = [ "examples" "doc" "out" ];
```

Comment:
Why can't the examples be placed at out?

Reply:
Done as suggested.